css/cssom-view/matchMedia.xht

```diff
@@ -1,188 +0,0 @@
-
-            test(function(){
-                assert_idl_attribute(mql, "matches", "Check that MediaQueryList.matches exists.");
-            }, "MediaQueryList.matches exists");
-
-            test(function(){
-                assert_readonly(mql, "matches", "Check that MediaQueryList.matches is readonly.");
-            }, "MediaQueryList.matches is readonly");
-
-            test(function(){
-                assert_true(mql.matches);
-            }, "MediaQueryList.matches for \"all\"");
-
-            test(function(){
-                assert_inherits(mql, "addListener");
-            }, "MediaQueryList.addListener exists");
-
-            test(function(){
-                assert_true(mql.addListener instanceof Function);
-            }, "MediaQueryList.addListener is a Function");
-
-            test(function(){
-                assert_inherits(mql, "removeListener");
-            }, "MediaQueryList.removeListener exists");
-
-            test(function(){
-                assert_true(mql.removeListener instanceof Function);
-            }, "MediaQueryList.removeListener is a Function");
-
-            test(function(){
-                mql = window.matchMedia("::");
-                assert_true(mql instanceof MediaQueryList, "window.matchMedia(\"::\") returned MediaQueryList object.");
-                assert_equals(mql.media, "not all", "MediaQueryList.media serialized as \"not all\" from original string with syntax error.");
-            }, "MediaQueryList.media syntax error");
-
-            test(function(){
-                assert_false(mql.matches);
-            }, "MediaQueryList.matches for \"not all\"");
-
-            test(function(){
-                mql = iframe_window.matchMedia("(max-width: 199px), all and (min-width: 200px)");
-                assert_equals(mql.media, "(max-width: 199px), (min-width: 200px)");
-                assert_true(mql.matches);
-            }, "MediaQueryList.matches for \"(max-width: 199px), all and (min-width: 200px)\"")
-
-            test(function(){
-                mql = iframe_window.matchMedia("(min-aspect-ratio: 1/1)");
-                assert_true(mql.matches);
-            }, "MediaQueryList.matches for \"(min-aspect-ratio: 1/1)\"");
-
-            test(function(){
-                mql = iframe_window.matchMedia("(width: 200px)");
-                assert_true(mql.matches);
-            }, "MediaQueryList.matches for \"(width: 200px)\"");
-
-            test(function(){
-                mql1 = iframe_window.matchMedia("(max-height: 50px)");
-                assert_false(mql1.matches);
-            }, "MediaQueryList.matches for \"(max-height: 50px)\"");
-
-            test(function(){
-                mql2 = iframe_window.matchMedia("(min-width: 150px)");
-                assert_true(mql2.matches);
-            }, "MediaQueryList.matches for \"(min-width: 150px)\"");
-
-            var resizeTest = async_test("Resize iframe from 200x100 to 200x50, then to 100x50");
```

Comment:
4 tests here (`removeListener` removes listener, resize calls listeners, listeners are called in order, `addListener` handles duplicates) were unnecessarily coupled, so I've carefully separated them and moved to [`MediaQueryList-addListener-removeListener.html`](https://github.com/web-platform-tests/wpt/pull/19841/files#diff-f9b9b87b535b01503769b492ffcc32b5)